xbmc/powermanagement/PowerManager.cpp

```diff
@@ -265,7 +278,7 @@ void CPowerManager::OnSleep()
 
 void CPowerManager::OnWake()
 {
-  CLog::Log(LOGNOTICE, "%s: Running resume jobs", __FUNCTION__);
+  CLog::Log(LOGNOTICE, "CPowerManager::OnWake(): Running resume jobs");
```

Comment:
Using hardcoded function names for log is error prone. I have seen more than once that people forgot to adjust this part when doing a refactoring of the code (rename functions, move code blocks etc.)

Reply:
that is the way to go for this PR. please no bikeshedding on a PR done by new contributor

Reply:
__FUNCTION__ doesn't have classname on linus

and 'CPowerManager:: + __FUNCTION__' result into 'CPowerManager::CPowerManager::OnWake()' in windows